tests/test_minimal/test_tools/aws_tools.py

```diff
@@ -35,14 +38,24 @@ def test_submit_aws_batch_job():
     time.sleep(60)
 
     job_id = info["job_submission_info"]["jobId"]
+    job = None
+    max_retries = 10
+    retry = 0
+    while retry < max_retries:
+        job_description_response = batch_client.describe_jobs(jobs=[job_id])
+        assert job_description_response["ResponseMetadata"]["HTTPStatusCode"] == 200
+
+        jobs = job_description_response["jobs"]
+        assert len(jobs) == 1
```

Comment:
This is a repeated example of the wait condition until a desired outcome is reached